build-linux-aarch64.sh

```diff
@@ -29,6 +31,11 @@ fi
 
 echo "ee3b0386d7a6474b042429e2fe7826fd40088258aec05707f0c722d773b5b1bd  packr_${PACKR_VERSION}.jar" | sha256sum -c
 
+# Note: Host umask may have checked out this directory with g/o permissions blank
+chmod -R u=rwX,go=rX appimage
```

Comment:
Have you tested this at all? I do not think there is a file here named `appimage` so I would expect this to fail

Reply:
This was required to build on my machine. `appimage` refers to https://github.com/runelite/launcher/tree/master/appimage

Reply:
Oh I see. I would prefer to just chmod the executable packr spits out similar to how the build-osx script works, than to modify packr itself. Can you just do that and close the packr pr?

Reply:
Oh sure I just thought it made sense to do it all in the one step. I might look at sending it upstream as welll because I'm sure others have run into the same snag before

Reply:
Hmm okay. We have a very old packr fork so I'm not sure how much it would apply to upstream or not.